git_practice.rb

```diff
@@ -1,3 +1,4 @@
-
+class Something
```

Comment:
Wow! Look great. The class is exactly what this needed.

Reply:
Thanks for the review!